File: cpp/Day05/demo04/student.cpp
#include "student.h"

Student::Student()
{
    this->rollno = 0;
    this->name = "";
    this->marks = 0;
}

Student::Student(int rollno, string name, double marks)
{
    this->rollno = rollno;
    this->name = name;
    this->marks = marks;
}

void Student::displayStudent()
{
    cout << "Rollno - " << rollno << endl;
    cout << "Name - " << name << endl;
    cout << "Marks - " << marks << endl;
}

void Student::acceptStudent()
{
    cout << "Enter the rollno - ";
    cin >> rollno;
    cout << "Enter the name - ";
    cin >> name;
    cout << "Enter the marks - ";
    cin >> marks;
}

void Student::setRollno(int rollno)
{
    this->rollno = rollno;
}
void Student::setName(string name)
{
    this->name = name;
}
void Student::setMarks(double marks)
{
    this->marks = marks;
}

int Student::getRollno()
{
    return rollno;
}
string Student::getName()
{
    return name;
}
double Student::getMarks()
{
    return marks;
}

File: java/Day04/Demo01/src/com/sunbeam/Student.java
package com.sunbeam;

import java.util.Scanner;

public class Student {
	private int rollno;
	private String name;
	private double marks;

	public Student() {
	}

	public Student(int rollno, String name, double marks) {
		this.rollno = rollno;
		this.name = name;
		this.marks = marks;
	}

	public int getRollno() {
		return rollno;
	}

	public void setRollno(int rollno) {
		this.rollno = rollno;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public double getMarks() {
		return marks;
	}

	public void setMarks(double marks) {
		this.marks = marks;
	}

	public void acceptStudent() {
		Scanner sc = new Scanner(System.in);
		System.out.print("Enter rollno - ");
		rollno = sc.nextInt();
		sc.nextLine();
		System.out.print("Enter name - ");
		name = sc.nextLine();
		System.out.print("Enter marks - ");
		marks = sc.nextDouble();
	}

	public void displayStudent() {
		System.out.println("Rollno - " + rollno);
		System.out.println("Name - " + name);
		System.out.println("Marks - " + marks);
	}

}
